weldx/time.py

```diff
@@ -155,6 +163,30 @@ class 'TimeDependent' are supported too.
     >>> t_res = "2000-01-01" + Time(["1d", "2d"])
     >>> t_res = ["3s", "4s"] + Time(["1s", "2s"])
 
+    Subtraction is possible too, but there are some restrictions. It is not possible
+    to subtract an absolute time from a time delta. Additionally, since the values of
+    a ``Time`` instance must be monotonically increasing, any subtraction that
+    produces a result that doesn't fulfill this requirement will fail. This is always
+    the case when subtracting arrays from scalars because either the array that is
+    subtracted (it is intrenally cast to a ``Time`` instance) or the resulting array
```

Comment:
```suggestion
    subtracted (it is internally cast to a `Time` instance) or the resulting array
```
just wondering, does the linking work here?